tests/MongoDB.Driver.Tests/Specifications/Runner/SessionsTestRunner.cs

```diff
@@ -0,0 +1,136 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.TestHelpers.JsonDrivenTests;
+
+namespace MongoDB.Driver.Tests.Specifications.Runner
+{
+    public abstract class SessionsTestRunner : BaseTestRunner
+    {
+        protected override string[] ExpectedTestColumns => new[] { "description", "clientOptions", "failPoint", "sessionOptions", "operations", "expectations", "outcome", "async" };
+
+        protected Dictionary<string, BsonValue> _sessionIdMap;
```

Comment:
yep, _objectMap is more general fashion, but there was a problem to get values from this collection outside of the `using` block since in this case, the session becomes disposed.

But I made small refactoring and currently additional step is not needed.